src/databricks/labs/ucx/hive_metastore/udfs.py

```diff
@@ -87,6 +87,9 @@ def _collect_tasks(self, catalog, database) -> Iterable[partial[Udf | None]]:
                     continue
                 udf_name = udf[udf.rfind(".") + 1 :]  # remove catalog and database info from the name
                 yield partial(self._describe, catalog, database, udf_name)
+        except NotFound:
+            # This make the integration test more robust as many test schemas are being created and deleted quickly.
+            logger.error(f"Schema {escape_sql_identifier(catalog)}.{escape_sql_identifier(database)} no longer existed")
```

Comment:
```suggestion
            logger.warning(f"Schema {escape_sql_identifier(catalog)}.{escape_sql_identifier(database)} no longer exist")
```
`logging.error` fails the workflow in the end

Reply:
you may want to do the same in `_describe`